### ISSUE TITLE
drawableLeft & drawableRight are not rendered

### ISSUE THREAD

**Issue by rockerhieu** [henrytao-me/android-md-core#108]
After upgrading to 3.3.2, drawableLeft & drawableRight are not rendered.

support library v24.1.1

``` xml
<TextView
        android:layout_width="wrap_content"
        android:layout_height="wrap_content"
        android:drawableLeft="@drawable/ic_wallet"
        android:drawablePadding="@dimen/spacing_small"/>
```

**Comment by rockerhieu** [comment]
Still happening in `3.3.4`

**Comment by henrytao-me** [comment]
Thanks. It's fixed in `3.3.5`. Let me know if you need further assist. 🙇

**Comment by rockerhieu** [comment]
Oh man, it was the fastest fix ever 🍻

**Comment by rockerhieu** [comment]
Yes, the issue is fixed with `3.3.5` 👯
